python/pyspark/sql/plot/core.py

```diff
@@ -388,6 +396,127 @@ def box(
         """
         return self(kind="box", column=column, precision=precision, **kwargs)
 
+    def kde(
+        self,
+        column: Union[str, List[str]],
```

Comment:
Create https://issues.apache.org/jira/browse/SPARK-49999 for a follow-up on optional "column" parameter support in box, kde and hist plots

Reply:
Create https://issues.apache.org/jira/browse/SPARK-50000 for  optional "bw_method" in both Pandas on Spark and PySpark